remoteappmanager/static/js/admin/vue-components/UsersView.js

```diff
@@ -0,0 +1,170 @@
+define([
+  "jquery",
+  "components/vue/dist/vue.min",
+  "jsapi/v1/resources"
+], function($, Vue, resources) {
+  "use strict";
+
+  Vue.component('new-user-dialog', {
+    template: `
+      <modal :show="show" :on-close="close">
+         <div class="modal-header"><h4>Create New User</h4></div>
+         <div class="modal-body">
+         <form>
+         <label for="user-name">User name</label>
+          <input type="text" class="form-control" id="user-name" v-model="name">
+          <div class="alert alert-danger" role="alert" v-show="name.length === 0">User name cannot be empty</div>
+         </form>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" @click="close()">Cancel</button>
+          <button type="button" class="btn btn-primary primary" :disabled="name.length === 0" @click="createNewUser">Create</button>
+        </div>
+    </modal>`,
+    props: ['show'],
+    data: function () {
+      return {
+        name: ''
+      };
+    },
+    methods: {
+      close: function () {
+        this.$emit('closed');
+      },
+      createNewUser: function() {
+        var user_name = $.trim(this.name);
```

Comment:
I don't understand why JQuery implements the trim function... It's equivalent to the Javascript method `this.name.trim()`, am I right ? So we should use the built in method instead of depending on a library for that.